script/99-imu-3dm-gx4.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEMS=="usb", ATTRS{manufacturer}=="Lord Microstrain",ATTRS{product}=="Lord Inertial Sensor", MODE="0777", GROUP="imu", SYMLINK+="lord_imu"
```

Comment:
Creating a symlink this way doesn't work well when you have multiple IMUs connected.